truss/tests/test_model_inference.py

```diff
@@ -193,6 +193,33 @@ def test_requirements_file_truss():
         assert response.json() is False
 
 
+@pytest.mark.integration
+def test_docker_server_truss():
+    with ensure_kill_all():
+        truss_root = Path(__file__).parent.parent.parent.resolve() / "truss"
+
+        truss_dir = truss_root / "test_data" / "test_docker_server_truss"
+
+        tr = TrussHandle(truss_dir)
+        LocalConfigHandler.set_secret("hf_access_token", "123")
+        _ = tr.docker_run(local_port=8090, detach=True, wait_for_server_ready=True)
+        truss_server_addr = "http://localhost:8090"
+        full_url = f"{truss_server_addr}/v1/models/model:predict"
+
+        try:
+            response = requests.post(full_url, json={})
+            print(f"response: {response}")
+        except requests.exceptions.ConnectionError:
+            pytest.fail("ConnectionError: Could not connect to the server")
```

Comment:
nit: I don't think this error message adds a whole lot over just throwing the exception, so I would consider just letting the exception get thrown. Up to you

Reply:
once wait_for_server_ready is fixed, we do not need this try catch any more